docs/how_to_guides/manifest_annotations.mdx

```diff
@@ -82,6 +82,12 @@ In order to add annotations using the `annotations.json`, you may run the follow
 oras push --annotation-file annotations.json localhost:5000/club:party cake.txt juice.txt
 ```
 
+This creates a manifest file. Running the following command to output the manifest:
```

Comment:
Maybe word this to align with the APIs. Also the file might not be  generated locally and an in memory implementation of the registry would not generate a file. So would not say file in either place. 

The push command would upload the blob and put a manifest. 
Running the …

Reply:
Ok @sajayantony, got it.